terraform/s3.tf.json.template.py

```diff
@@ -1,9 +1,32 @@
-import boto3
-import json
 from azul import config
-from azul.deployment import aws
 from azul.template import emit
 
+# Route 53 records are only created for dev environments (not staging, intergration, or prod)
+data = {
```

Comment:
I don't think this extracted variable adds any clarity. Quite the opposite, actually.

The alternative is to extract the entire argument do emit() into a variable and then use `if` statements to conditionally modify it before its passed to emit(). But can we stick with precedent here and just inline the variable?